### PR TITLE
[#678] NUCATS Grants Repository Changes

### LINKED ISSUE #678: Update NUCATS Grants Repository collection
The following updates to be made to the institutional collection "NUCATS Grants Repository" by Wednesday, August 14, 2019.

New Collections, all "Northwestern University" visibility: 
1. Pink Sheets, under Grant Resources collection
2. Biosketches, under Grant Resources collection
3. Federal Non-NIH Awards
4. R-Series
5. U-Series

Move "NUCATS Grants Repository" as a child collection of new institutional collection "NUCATS Center for Education and Career Development", Private visibility.

### PR DESCRIPTION
**Pull request by sharpattack**
New Parent Institutional Collection
- NUCATS Center for Education and Career Development
- Move NUCATS Grants Repository into this collection

New child collections for NUCATS Grants Repository
- Federal Non-NIH
- U-series
- R-series
- Grants Resources
  - Pink Sheets
  - Biosketches

closes #678

### CODE REVIEW

File: lib/batch/institutional_nucats_cecd.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# https://github.com/galterlibrary/digital-repository/issues/678
+# Create new parent collection 'NUCATS Center for Education and Career Development'
+# move NUCATS Grants Repository as child
+
+# NUCATS Grants Repository
+parent_nucats_grants = Collection.find("f2bf6e1d-0e32-4ce2-a52e-bb0522d5708d")
+
+admin_group = 'NUCATS-CECD-System-Admin'
+depositor = 'institutional-nucats-cecd'
```
Comment (fenekku) [inline lib/batch/institutional_nucats_cecd.rb]: `depositor` isn't used in this script nor is the value `'institutional-nucats-cecd'`, but `parent_nucats_grants.depositor` is used in the other script. Should `depositor` just be removed here or is its value the one that should be used for `nucats_cecd_sys_user.username`?

File: lib/batch/institutional_nucats_grants_repo_child_collections.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+parent_nucats_grants.members << r_series 
+parent_nucats_grants.save!
+
+# Add R01, R03, R21 collections
+# note: removing these collections from NUCATS Grants Repository collection 
+# can be done on the frontend
+r01 = Collection.find("6a9690f5-664a-4654-a5ca-c1c1ca99db9b")
+r03 = Collection.find("49945521-3b35-4efd-9e77-e909900f1604")
+r21 = Collection.find("af85d02b-4b6c-48db-8cac-b82f4075e1f0")
+r_series.memebers << r01
```
Comment (fenekku) [inline lib/batch/institutional_nucats_grants_repo_child_collections.rb]: `memebers` ! 

slowpoke_meme
(see urban dictionary's defintion of a **memeber** at your own risk)

File: lib/batch/institutional_nucats_cecd.rb (hunk 1 above)
Comment (sharpattack) [reply lib/batch/institutional_nucats_cecd.rb]: Thanks for catching that. The value is used for converting the collection to institutional.

File: lib/batch/institutional_nucats_grants_repo_child_collections.rb (hunk 2 above)
Comment (sharpattack) [reply lib/batch/institutional_nucats_grants_repo_child_collections.rb]: haha, nice catch.

**Comment by sharpattack** [comment]
@fenekku please review the latest changes, as they were needed after testing on stage.

**Pull request merged**